affiliate/package.json

```diff
@@ -21,6 +21,9 @@
 		"eslint-plugin-react": "7.30.0"
 	},
 	"dependencies": {
+		"@wordpress/block-editor": "^10.5.0",
+		"@wordpress/blocks": "^11.21.0",
+		"@wordpress/components": "^22.1.0",
```

Comment:
Added these so they could join `@wordpress/element` in being explicitly defined.